### ISSUE TITLE
Incoherent behavior with drill-down dimensions

### ISSUE THREAD

**Issue by LaurentBerder** [Parkman328/SenseSankey#12]
Hi,

I used SenseSankey (thanks for your awesome work) with a drill-down dimension for the origin, and another drill-down dimension for the destination.
When I drill down on the destination, I can still see the upper level of the origin dimension on the left. However, if I drill down on the origin, the destination dimension is also automatically drilled down.
This destination dimension of my sankey diagram also gets drilled down to lower level if I filter on any other graph.

Basically, the only way I can see the higest level of my drill-down destination dimension on the sankey diagram is by using no filter at all.

Any idea why this would happen?

**Comment by LaurentBerder** [comment]
Here's my example with no filter (there's only one country for the destination, which is Brasil) ![sankey no filter](https://user-images.githubusercontent.com/19341543/35432837-a59a1a60-0281-11e8-810f-c72d8ba51898.PNG)

If I drill down on the destination, I get the detail of ports in Brasil, but still have the high level of world regions on the origin dimension on the left: 
![sankey drilled down destination](https://user-images.githubusercontent.com/19341543/35432902-de34c1a4-0281-11e8-87ea-9a98460b7d76.PNG)

But if I go back to the start and filter on the origin (here I selected the region Northern Europe, getting European countries), the destination also gets drilled to the port level instead of showing only Brasil: 
![sankey drilled down origin](https://user-images.githubusercontent.com/19341543/35432970-1d97af46-0282-11e8-86a7-5aaa46eec0f9.PNG)
